SEDaily-IOS/AudioView.swift

```diff
@@ -78,6 +78,8 @@ public protocol AudioViewDelegate: NSObjectProtocol {
 }
 
 class AudioView: UIView {
+    static var userSettingPlaybackSpeedKey = "PlaybackSpeed";
```

Comment:
Being pedantic but this could live in the manager too as nothing else in the view requires it :)